python/delta-kernel-python/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "delta-kernel-python"
+edition = "2021"
```

Comment:
what's 2021 for? should it be 2024?

Reply:
yeah Ill change it, this came from the hackathon branch and I forgot to take a close look into the maturin and rust components

Reply:
nvm i cant change it, this is a edition that specifies some build configurations. When i changed it to 2024 it crashed the maturin develop